src/hooks/useNewFolderDialog.ts

```diff
@@ -58,10 +57,7 @@ export default function useNewFolderDialog() {
       );
       try {
         await addNewFolder();
-        await handleFileBrowserNavigation({
-          fspName: currentFileSharePath.name,
-          path: currentFileOrFolder.path
-        });
+        await fetchFiles(currentFileSharePath.name, filePath);
```

Comment:
For whatever reason, when I add a new folder, I see the toast to tell me that the folder was added, but I am not seeing it in the list, until I refresh the page.